### PR TITLE
CM-361: DateTimeOffset vs DateTime in C#

### PR DESCRIPTION
**Pull request by ralphtashinganyoni**

### CODE REVIEW

File: dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime/Program.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+DateTime dateTime = DateTime.Now; // Represents the current local time
+Console.WriteLine($"DateTime: {dateTime}"); // DateTime: 6/28/2023 11:01:34 PM
+
+DateTimeOffset dateTimeOffset = DateTimeOffset.Now; // Represents the current local time with offset information
+Console.WriteLine($"DateTimeOffset: {dateTimeOffset}"); // DateTimeOffset: 6/28/2023 11:01:34 PM +02:00
+
+//TimeZone vs Offset 
+static List<TimeZoneInfo> GetTimeZoneFromOffset(TimeSpan offset)
+{
+    List<TimeZoneInfo> list = new List<TimeZoneInfo>();
```
Comment (PhilBroderick) [inline dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime/Program.cs:13]: We can use `var` here.

File: dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime/Program.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Console.WriteLine($"DateTime Kind: {dateTimeUnspecified.Kind}"); // DateTime Kind: Unspecified
+
+
+
+DateTimeOffset dateTimeOffsetUnspecified = DateTimeOffset.Now; // Represents the current time
+Console.WriteLine($"DateTimeOffset Kind: {dateTimeOffsetUnspecified.DateTime.Kind}"); // DateTime Kind: Unspecified
+
+
+
+
```
Comment (PhilBroderick) [inline dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime/Program.cs:48]: Some extra spacing here that is unnecessary.

File: dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime.Tests/DateTimeOffsetVsDateTimeTest.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
+using DateTimeOffsetVsDateTime;
+namespace DateTimeOffsetVsDateTime.Tests;
+
+public class DateTimeOffsetVsDateTimeTest
```
Comment (PhilBroderick) [inline dotnet-datetime/DateTimeOffsetVsDateTime/DateTimeOffsetVsDateTime.Tests/DateTimeOffsetVsDateTimeTest.cs:4]: A lot of these test cases can be simplified into 1 liners, without the variable declaration.

**Comment by jshergal** [comment]
Also worthy of note here is that this PR has a lot of commits unrelated to the article. If we check the author guidelines under [implementation code](https://docs.google.com/document/d/1PRj3IPPPNTF-NU-XGQqthflVd3m8tkX7c2JGn4gfX74/edit) we see the following:

> the Pull Request (PR)
> - when your code is ready, create a Pull Request (PR) from your fork to our repository - for your editor to review and merge
> - it’s worth mentioning that each article code should be created on a separate branch on your fork and then the PR should be created from that branch to the repository main
> - this way we get a clean commit history instead of dragging old commits along

Also of note in the guideline are these notes:

> - note 1: try to keep the PR under 10 commits so that we can merge them quickly
> - note 2: all PRs should contain the JIRA issue number in the title. Example: CM-234: Fixing typos

**Pull request closed without merging**